lua/octo/navigation.lua

```diff
@@ -31,11 +31,11 @@ function M.open_in_browser(kind, repo, number)
       return
     end
     if buffer:isPullRequest() then
-      cmd = string.format("gh pr view --web -R %s %d", buffer.repo, buffer.number)
+      cmd = string.format("gh pr view --web %d", buffer.number)
```

Comment:
Will this work for GitHub.com though? I guess it will if your CWD is in the repo you want to browse, but will probably fail otherwise

Reply:
Hm... I'm never tried to use it this way, will return later with changes. Thanks for notice